core/src/main/java/org/everit/json/schema/NumberSchema.java

```diff
@@ -150,7 +150,7 @@ public boolean requiresInteger() {
         return requiresInteger;
     }
 
-    public boolean isRequiresNumber() {
+    public boolean requiresNumber() {
```

Comment:
Please don't rename public methods, it breaks API compatibility (I know that `isRequiresNumber()` is a very poor name :))